src/main/java/io/mailtrap/model/request/contactexports/ContactExportFilter.java

```diff
@@ -0,0 +1,28 @@
+package io.mailtrap.model.request.contactexports;
+
+import lombok.Getter;
+
+import static io.mailtrap.model.request.contactexports.ContactExportFilterName.list_id;
+import static io.mailtrap.model.request.contactexports.ContactExportFilterName.subscription_status;
+
+@Getter
+public class ContactExportFilter {
+
+    private final String name;
+    private final String operator;
+    private final Object value;
+
+    private ContactExportFilter(final String name, final String operator, final Object value) {
+        this.name = name;
+        this.operator = operator;
+        this.value = value;
+    }
+
+    public static ContactExportFilter listIDs(final ContactExportFilterOperator operator, final Long... listIDs) {
+        return new ContactExportFilter(list_id.name(), operator.getValue(), listIDs);
+    }
```

Comment:
I just noticed that in other places in this code the convention is `Id` and not `ID`. Sorry for suggesting `ID` before


```suggestion
    public static ContactExportFilter listIds(final ContactExportFilterOperator operator, final Long... listIds) {
        return new ContactExportFilter(list_id.name(), operator.getValue(), listIds);
    }
```